### DISCUSSION TITLE
Comets 29P and 62P wrong position in Stellarium

### DISCUSSION THREAD

**Discussion by Sternefueralle2023** [Stellarium/stellarium#3510] [General]
Why are the two comets in Stellarium that meet closely on Wednesday into Thursday in the wrong position: 29P and 62P?

**Comment by alex-w** [comment]
Please check orbital elements for both

**Reply by Sternefueralle2023** [reply]
I imported the two comets into Stellarium, but they are in the wrong place. Normally something like this works.

**Reply by gzotti** [reply]
How wrong is "wrong"? Outdated elements even just a few weeks old can easily cause arcminutes or more of deviation. You may need to open (in a text editor) the ssystem_minor.ini and check numbers for that.

**Reply by Sternefueralle2023** [reply]
The comets 62P and 29P are on the night of November 15th to 16th. close to each other. At this point in time, however, in Stellarium it says 62P in the SW and 29P in the East. There's something wrong with Stellarium, right?

**Reply by gzotti** [reply]
I don't know the "real" or "correct" numbers. 4 arcminutes separation are pretty close IMHO. It all depends on the orbital elements. This is with data from "MPC's list of observable comets".
![stellarium-109](https://github.com/Stellarium/stellarium/assets/18099873/feac2dff-6e62-4ee2-9975-a07535845aab)
